qemu/tests/cfg/create_large_raw_img.cfg

```diff
@@ -0,0 +1,21 @@
+- create_large_raw_img:
+    only raw
+    virt_test_type = qemu
+    type = create_large_raw_img
+    start_vm = no
+    create_image = no
+    remove_image = yes
+    images = "filename loop"
```

Comment:
`filename` is misleading.

Reply:
Maybe you could list all of advises in one time, please. Or I have to push and re-test the case for a few times.